src/main/java/io/github/railroad/Railroad.java

```diff
@@ -23,8 +23,17 @@ public class Railroad extends Application {
     public static final Gson GSON = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
     private static boolean DEBUG = false;
 
-    public static Scene SCENE;
-    public static Stage WINDOW;
+    private static Scene _scene;
+    private static Stage _window;
```

Comment:
Why have you put `_`s before them 😭. We don't do that in java.

Reply:
😭 i'm giving up java officially